.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
looks like this file shouldn't be here

Reply:
Yup. Probably best to add `.vscode` to `.gitignore` so this and related files are ignored going forward.

Reply:
Yes, that is true. Thank you for pointing this out as well!